contracts/contracts/token/WOETH.sol

```diff
@@ -106,87 +113,8 @@ contract WOETH is ERC4626, Governable, Initializable {
 
```

Comment:
transferToken should use the generic "core asset" in both the require and the comment, so that it can be used unchanged in other code bases.